docs/api/ShallowWrapper/exists.md

```diff
@@ -14,5 +14,5 @@ Returns whether or not the current node exists.
 
 ```jsx
 const wrapper = shallow(<div className="some-class" />);
-expect(wrapper.find('.other-class').exists()).to.be(false);
```

Comment:
This is actually correct as-is; we use chai's expect, not jest's; separately, no-op matchers (ie, any matcher that isn't a function call) are extremely dangerous, because `expect(value).to.be.fals` or `expect(value).to.be.yogurt` will both silently fail even though those matchers don't exist.

Reply:
I am using chai's `expect` as well, not using jest, and `.to.be()` throws with `.to.be is not a function` and as much as I searched I could not find any part of the documentation where chai's `.be` is used as a function. 

I think something from your reply must be going over my head :/

Reply:
hmmmm good call, you're right. however, `.to.be.false` isn't the right fix :-)

Reply:
ok so, you're totally right :-) let's change this to `.to.equal(false)`

Reply:
Done!